packages/winpanda/extra/src/winpanda/tests/unit/test_cfgm_cfgm.py

```diff
@@ -0,0 +1,24 @@
+import mock
+import unittest
+import pytest
+
+from cfgm.cfgm import PkgConfManager
+from cfgm import exceptions
+from core.package.manifest import PackageManifest
+
+
+class TestPkgConfManager(unittest.TestCase):
+    """DC/OS package configuration files manager unit tests."""
+
+    @mock.patch('cfgm.cfgm.isinstance', return_value=True)
+    def test_init_should_check_manifest_type(self, *args):
+        """Check PkgConfManager __str__ magic method return value."""
+        manifest = mock.Mock()
+        cfg_manager = str(PkgConfManager(manifest))
+        assert isinstance(cfg_manager, str)
+
+    def test_setup_empty_manifest_should_fail(self):
```

Comment:
What is an "empty mainfest"? Is it possible to create one without using a mock? Again, this seems like we could simplify by using mypy.

Reply:
Agree, in a case mypy we will check strict typing and after inplementation we can remove this test.